USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -0,0 +1,250 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using USFMToolsSharp.Models.Markers;
+using NPOI.XWPF.UserModel;
+using System.IO;
+
+namespace USFMToolsSharp.Renderers.Docx
+{
+    public class DocxRenderer
+    {
+        public List<string> UnrenderableTags;
+        public Dictionary<string,List<Marker>> FootnoteTextTags;
+        
+        private XWPFDocument newDoc;
+
+        private bool isSingleSpaced = true;
+        private bool hasOneColumn = true;
+        private bool isL2RDirection = true;
+        private bool isTextJustified = false;
+        private bool separateChapters = true;
+
+        public string FrontMatterDocx { get; set; }
+        public string InsertedFooter { get; set; }
+        public string InsertedHead { get; set; }
+
+        public DocxRenderer()
+        {
+            UnrenderableTags = new List<string>();
+            FootnoteTextTags = new Dictionary<string,List<Marker>>();
+            newDoc = new XWPFDocument();
+        }
+        public DocxRenderer(bool isSingleSpaced = true, bool hasOneColumn = true, bool isL2RDirection = true, bool isTextJustified = false, bool separateChapters = false)
```

Comment:
If we're going to do a configuration object for the html renderer it would make logical sense for it to be present in this one too

Reply:
I still am looking up how to manipulate the styling using NPOI. I'm not finding much for columns